behave_testrail_reporter/testrail_reporter.py

```diff
@@ -39,15 +39,21 @@ def format_summary(statement_type, summary):
 
 
 class TestrailProject(object):
-    def __init__(self, id, name, suite_id, test_run_name='', allowed_branch_pattern='*'):
+    def __init__(self, id, name, suite_id, allowed_branch_pattern='*'):
         self.id = id
         self.name = name
         self.suite_id = suite_id
-        self.test_run_name = test_run_name
         self.allowed_branch_pattern = allowed_branch_pattern
         self.test_run = None
         self.cases = {}
 
+    def get_test_run_name(self, branch_name):
+        return self.name.format(
+            project_id=self.id,
+            suite_id=self.suite_id,
+            name=self.name,
```

Comment:
`name` is in here, but never tested or mentioned in the readme. Does it need to be there?

Reply:
probably now it does not since it's the name itself, was something that was left by mistake when I was exploring having a `name_pattern` config option. will remove it